pycsw/core/metadata.py

```diff
@@ -1102,6 +1105,6 @@ def caps2iso(record, caps, context):
     apiso_obj = APISO(context.model, context.namespaces, context)
     apiso_obj.ogc_schemas_base = 'http://schemas.opengis.net'
     apiso_obj.url = context.url
-    queryables = dict(apiso_obj.repository['queryables']['SupportedISOQueryables'].items() + apiso_obj.repository['queryables']['SupportedISOQueryables'].items())
+    queryables = dict(list(apiso_obj.repository['queryables']['SupportedISOQueryables'].items()) * 2)
```

Comment:
Looks like a vintage copy/paste typo :)

Having said this, if we take it out, the tests will tell us whether this matters or not.